plan/plan_test.go

```diff
@@ -69,6 +79,14 @@ func TestCalculate(t *testing.T) {
 		{[]Policy{&SyncPolicy{}}, labeledV1, noLabels, empty, labeledV1, labeledV2, empty},
 		// RecordType should be inherited
 		{[]Policy{&SyncPolicy{}}, typedV1, noType, empty, typedV1, typedV2, empty},
+		// If desired TTL is not configured, do not update
+		{[]Policy{&SyncPolicy{}}, ttlV1, ttlV2, empty, empty, empty, empty},
+		// If desired TTL is configured but is the same as current TTL, do not update
+		{[]Policy{&SyncPolicy{}}, ttlV1, ttlV3, empty, empty, empty, empty},
+		// If desired TTL is configured and is not the same as current TTL, need to update
+		{[]Policy{&SyncPolicy{}}, ttlV1, ttlV4, empty, ttlV1, ttlV4, empty},
+		// If target changed and desired TTL is not configured, do not update TTL
+		{[]Policy{&SyncPolicy{}}, ttlV1, ttlV5, empty, ttlV1, ttlV6, empty},
```

Comment:
Yeah, I think plan's purpose should just make sure to make `ttlV1` to look like `ttlV5`. If `ttlV5` has a weird TTL defined, so be it.

Reply:
This is outdated, ignore it.